project/Dependencies.scala

```diff
@@ -163,8 +163,10 @@ object Dependencies {
 
   val AvroParquet = Seq(
     libraryDependencies ++= Seq(
-        "org.apache.parquet" % "parquet-avro" % "1.10.1" //Apache2
-      )
+        "org.apache.parquet" % "parquet-avro" % "1.13.1", //Apache2
+        "org.apache.avro" % "avro" % "1.11.3"
```

Comment:
Is it ok to add the explicit override? It evicts `1.11.1` brought in by `avro-parquet`.

Reply:
Put in a comment that it was added to bump the version brought in by `avro-parquet`.